_includes/anchor_links.html

```diff
@@ -0,0 +1,45 @@
+<script type="text/javascript">
+  var anchorForId = function (id)
```

Comment:
Tabs for indentation, please :)

Also, this should follow the [WordPress JS coding standards](https://make.wordpress.org/core/handbook/coding-standards/javascript/) in general, with the opening braces on the same line as the rest.